pkg/version/version.go

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package version
+
+var (
+	Version = "0.0.0-master"
```

Comment:
I don't understand the version format here. Why `master` and why `0.0.0`? Should it be `0.1.0 `?

Reply:
@fpj this is just a placeholder for the version. The initial `0.0.0-master` just indicates that we have not released any version yet and we are based on master for the moment. For the first release, this field will be updated to `0.0.1`.

Reply:
@maddisondavid thanks for the feedback. I'll explore that alternative.